src/components/shared/DownloadButton.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from "react";
+import Translate from "../../utils/translate/translateService";
+import { sendAnalyticsEvent } from '../../utils/analyticsUtils';
+import { Button } from "semantic-ui-react";
+
+interface DownloadButtonProps {
+    buttonLabelKey?: string;
+    downloadLink?: string;
```

Comment:
PS I went through the effort of making this component a bit more customizable because we might use a similar pattern when implementing the button used to access our data via github